Link against libtirpc variant built without GSS-API/Kerberos to avoid
unnecessary dependency on krb5-libs (open-vm-tools don't use Kerberos).

--- a/open-vm-tools/configure.ac
+++ b/open-vm-tools/configure.ac
@@ -1016,9 +1015,9 @@
 
 
 if test "$with_tirpc" != "no"; then
-   AC_VMW_CHECK_LIB([libtirpc],
+   AC_VMW_CHECK_LIB([libtirpc-nokrb],
                     [TIRPC],
-                    [libtirpc],
+                    [libtirpc-nokrb],
                     [],
                     [],
                     [],
